sizzlingstats/serverplugin_empty.cpp

```diff
@@ -841,6 +844,26 @@ void CEmptyServerPlugin::FireGameEvent( IGameEvent *event )
 			}
 		}
 	}
+	else if ( m_bShouldRecord && FStrEq( name, "arrow_impact" ) )
+	{
+		if (event->GetInt("projectileType") == 11) // crusader's crossbow

```

Comment:
const int sCrusadersCrossbowType = 11;
instead of just the 11 and a comment.

Same with the logstats file above